books/free-programming-books-subjects.md

```diff
@@ -655,6 +656,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 * [Mininet Walkthrough](https://mininet.org/walkthrough/)
 * [Network Science](http://networksciencebook.com) - Albert-Laszló Barabási
 * [Networking! ACK!](https://wizardzines.com/zines/networking/) - Julia Evans (PDF)
+* [Networking Fundamentals](https://www.haio.ir/app/uploads/2022/02/Networking-Fundamentals-by-Crystal-Panek-z-lib.org_.pdf) - Crystal Panek (PDF)
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.
remove it

Reply:
This is my first time submitting a pull request, I'm not quite familiar with the process. What is the issue on this link? What does it mean that its not an authorized distribution? How do I remove it? Thank you for reviewing!

Reply:
it mean we will not accept it.